### ISSUE TITLE
row_order and column_order changes

### ISSUE THREAD

**Issue by RFSH** [informatics-isi-edu/ermrestjs#647]
Recently I changed row_order to apply the column_order logic too. That means I am going through the list of `row_order` columns and if they have `column_order` defined, I am using those columns instead of the column specified in the `row_order`. We should revert that and use the `row_order` as is.

We also need to unify the syntax of `row_order` and `column_order`. Current syntax of `row_order` requires an object in `{"column": COL_NAME, "descending": BOOLEAN}` format, and `column_order` just accepts column name. We should unify them so,

for `row_order`:
  - `"row_order": ["col"]` should be treated the same way as `"row_order": [{"column": "col"}]`.

for `column_order`:
  - `"column_order": [{"column": "col"}]` should be treated the same as `"column_order": ["col"]`.
  - `"column_order": [{"column": "col", "descending": true}]` means that we have to revert the sort order.

tl;dr
- [ ] Revert row_order change to just use the list as is (without applying column_order logic).
- [ ] Unify the column_order and row_order syntax.
